src/js/jsx/shared/Select.jsx

```diff
@@ -47,12 +47,13 @@ define(function (require, exports, module) {
             value: React.PropTypes.shape({
                 id: React.PropTypes.string.isRequired,
                 title: React.PropTypes.string.isRequired,
-                style: React.PropTypes.object
+                info: React.PropTypes.string,
+                style: React.PropTypes.object,
+                svgType: React.PropTypes.string
```

Comment:
I don't think it can be added as part of `style` because I think that's just CSS being passed in.

Before I got to this, it had `id`, `value`, and `style` grouped under `value`. I'm not sure what makes those values different from the other 3 that an `Option` has (`selected`, `next`, and `prev`), so I don't know what the point would be of making them into a separate ReactElement.